data/assets/scene/solarsystem/dwarf_planets/pluto/pluto.asset

```diff
@@ -51,9 +51,9 @@ local PlutoLabel = {
   Renderable = {
     Enabled = false,
     Type = "RenderableLabels",
-    Size = "Pluto",
```

Comment:
Oopsiedupsie

Reply:
😅 Good that I had to go through all the labels once more